src/Reboot/OnExceptionRebootStrategy.php

```diff
@@ -86,7 +86,7 @@ public function clear(): void
         $this->forceRebootEventCaught = null;
     }
 
-    public static function getSubscribedEvents()
+    public static function getSubscribedEvents(): array
```

Comment:
Thanks, can you also add the type in [`SentryListener`](https://github.com/Baldinof/roadrunner-bundle/blob/2.x/src/Integration/Sentry/SentryListener.php#L35) please?

Reply:
Done :)